App.tsx

```diff
@@ -1,23 +1,18 @@
 import React from 'react'
-import { SafeAreaView, StyleSheet } from 'react-native'
+import { NavigationContainer } from '@react-navigation/native'
 
-import { COLORS, SPACING } from './src/theme'
-import { DActive, Notifications } from './src/screens'
+import { StackNavigator } from './src/navigators'
+import { Provider } from 'react-redux'
+import { store } from './src/redux/store'
 
 const App = () => {
-  const showNotificationScreen = false
   return (
-    <SafeAreaView style={styles.container}>
```

Comment:
this should not be removed. Please read https://reactnative.dev/docs/safeareaview.
Just wrap it around the main App.tsx, no need to add it on every screen.